Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis.yml

```
@@ -0,0 +1,93 @@
+  description: Identifies lateral movement techniques in the original command line.
+- contextPath: CommandLineAnalysis.analysis.original.data_exfiltration
+  description: Detects data exfiltration activities in the original command line.
+- contextPath: CommandLineAnalysis.analysis.original.amsi_techniques
+  description: Checks for AMSI bypass techniques in the original command line.
+- contextPath: CommandLineAnalysis.analysis.original.indicators
+  description: Extracts indicators of compromise (IOCs) from the original command line.
+- contextPath: CommandLineAnalysis.analysis.original.base64_encoding
+  description: Decoded content from Base64 encoding in the original command line.
+- contextPath: CommandLineAnalysis.analysis.original.reversed_command
+  description: Indicates if the original command line was reversed.
+- contextPath: CommandLineAnalysis.analysis.decoded.malicious_commands
+  description: Checks for malicious commands in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.windows_temp_path
+  description: Checks if the decoded Base64 command line accesses Windows temporary paths.
+- contextPath: CommandLineAnalysis.analysis.decoded.suspicious_parameters
+  description: Identifies suspicious parameters or content in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.mixed_case_powershell
+  description: Detects mixed case PowerShell commands in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.powershell_suspicious_patterns
+  description: Searches for suspicious PowerShell patterns in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.credential_dumping
+  description: Checks for credential dumping techniques in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.custom_patterns
+  description: Matches custom patterns (if provided) in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.reconnaissance
+  description: Checks for reconnaissance activities in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.lateral_movement
+  description: Identifies lateral movement techniques in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.data_exfiltration
+  description: Detects data exfiltration activities in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.amsi_techniques
+  description: Checks for AMSI bypass techniques in the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.analysis.decoded.indicators
+  description: Extracts indicators of compromise (IOCs) from the decoded Base64 command line.
+- contextPath: CommandLineAnalysis.Double Encoding Detected
+  description: Identifies nested Base64 strings.
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/python3:3.12.7.117934
```

Due to a technical issue with python 3.12 support in the build, please change the docker image to the lastest python 3.11 docker image tag.
```suggestion
dockerimage: demisto/python3:3.11.10.116949
```